daemons/gptp/common/ptp_message.cpp

```diff
@@ -933,7 +937,7 @@ void PTPMessageFollowUp::processMessage(IEEE1588Port * port)
 	sync_arrival = sync->getTimestamp();
 
 	delay = port->getLinkDelay();
-	if ((delay = port->getLinkDelay()) == 3600000000000) {
+	if ((delay = port->getLinkDelay()) == 3600000000000) { /*TODO: ONE_WAY_DELAY_DEFAULT*/
```

Comment:
That would make this more clear. The other possibility would be to add a boolean such as link_delay_valid or something like that.

bool link_delay_is_valid = port->getLinkDelay( &delay );
if( !link_delay_valid ) {
  goto done;
}

// Else continue processing message

The port code may already have a link delay valid flag.